musl no-ops sched_setscheduler/posix_spawnattr_setschedparam so you're forced to
use the pthread method.
--
--- a/src/utils/realtime.cpp
+++ b/src/utils/realtime.cpp
@@ -8,6 +8,7 @@
 
 #include "config-kwin.h"
 
+#include <pthread.h>
 #include <sched.h>
 
 namespace KWin
@@ -19,7 +20,7 @@
     const int minPriority = sched_get_priority_min(SCHED_RR);
     sched_param sp;
     sp.sched_priority = minPriority;
-    sched_setscheduler(0, SCHED_RR | SCHED_RESET_ON_FORK, &sp);
+    pthread_setschedparam(pthread_self(), SCHED_RR | SCHED_RESET_ON_FORK, &sp);
 #endif
 }
 
